src/linear_algebra/clifford_algebra/basic.lean

```diff
@@ -313,6 +314,9 @@ def invertible_ι_of_invertible (m : M) [invertible (Q m)] : invertible (ι Q m)
   mul_inv_of_self := by rw [map_smul, mul_smul_comm, ι_sq_scalar, algebra.smul_def, ←map_mul,
     inv_of_mul_self, map_one] }
 
+def invertible_of_invertible_ι (m : M) [invertible (ι Q m)] [invertible (2 : R)] :
+  invertible (Q m) := sorry
```

Comment:
Any more ideas on how to prove this or under what situations it's true?

Reply:
I already got the proof of this. I just need some time to tidy it up.

Reply:
Could you commit the messy version somewhere so that it doesn't get lost?

Reply:
> Could you commit the messy version somewhere so that it doesn't get lost?

@Biiiilly Hi, I'm trying to make this PR work in Lean 4, do you still have the proof of this?